content/guides/tina-cloud/getting-started/defining-content.md

```diff
@@ -49,31 +46,44 @@ As you can see, we have a quite a few fields that we want our content team to be
 
 ### Making changes to the Schema
 
-Open up the Tina `schema.ts` file located at `/.tina/schema.ts` To begin with underneath the object we provided, we need to create a new collection object:
+Open up the Tina `schema.ts` file located at `/.tina/schema.ts` To begin with underneath the object we provided, we need to replace the current collection with the content we want:
 
 ```js
```

Comment:
Should this be diff and not JS?

Reply:
Yup should be diff. I did so many changes forgot to go back through!